docs/2_step_by_step/2_2_experiment_and_run.md

```diff
@@ -1,11 +1,11 @@
 (experiment-and-runs)=
 # Two basic concepts: Experiment and Run
 
-Finetuner organizes your training based on two concepts: `Experiment` and `Run`.
+Finetuner organizes your training based on two concepts: {class}`~finetuner.experiment.Experiment` and {class}`~finetuner.run.Run`.
 
-An `Experiment` defines the machine learning task you're fine-tuning for.
-A `Run` is a piece of code that performs the `Experiment` with specific configurations.
-An `Experiment` contains a list of `Run`s,
+An {class}`~finetuner.experiment.Experiment` defines the machine learning task you're fine-tuning for.
```

Comment:
```suggestion
An Experiment defines the machine learning task you're fine-tuning for.
```
There is no need to link to the source every time you mention a class, I think it becomes a bit tiring on the eyes actually. After you have used the link once, just upper case the word (also don't use `backticks`, it becomes a bit tiring).